model_evaluation/Eval Notebook.ipynb

```diff
@@ -0,0 +1,2082 @@
+{
```

Comment:
Can you rename file to contains "biosbias"?

Reply:
Done.